### PR TITLE
SORQA-272

### PR DESCRIPTION
**Pull request by pk-sgent**
SORQA-272

### CODE REVIEW

**Comment by pk-sgent** [comment]
https://github.com/hzi-braunschweig/SORMAS-Project/pull/9413

**Comment by pk-sgent** [comment]
image

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java [hunk 1]
```diff
@@ -1622,6 +1623,14 @@ public EditCaseSteps(
           webDriverHelpers.clickOnWebElementBySelector(ACTION_CONFIRM);
           webDriverHelpers.waitForPageLoadingSpinnerToDisappear(40);
         });
+
+    When(
+        "I check if Type of sample has not a ([^\"]*) option",
+        (String option) -> {
+          softly.assertFalse(
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditCaseSteps.java:1630]: add assert message

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditContactsSteps.java [hunk 2]
```diff
@@ -335,6 +337,18 @@ public EditContactsSteps(
         () -> {
           webDriverHelpers.clickOnWebElementBySelector(getContactByUUID(contactsUUIDList.get(1)));
         });
+
+    When(
+        "I check if Task Type has not a ([^\"]*) option",
+        (String option) -> {
+          softly.assertFalse(
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EditContactsSteps.java:344]: add assert message

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EpidemiologicalDataCaseSteps.java [hunk 3]
```diff
@@ -644,6 +645,21 @@ public EpidemiologicalDataCaseSteps(
               "Exposure descriptions are not equal");
           softly.assertAll();
         });
+
+    When(
+        "I check if Exposure Type of activity has not a ([^\"]*) option",
+        (String option) -> {
+          softly.assertFalse(
+              webDriverHelpers.checkIfElementExistsInCombobox(TYPE_OF_ACTIVITY_COMBOBOX, option));
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EpidemiologicalDataCaseSteps.java:653]: same

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EpidemiologicalDataCaseSteps.java [hunk 4]
```diff
@@ -644,6 +645,21 @@ public EpidemiologicalDataCaseSteps(
...
+          softly.assertAll();
+        });
+
+    When(
+        "I check if Exposure details has a ([^\"]*) option",
+        (String option) -> {
+          softly.assertFalse(webDriverHelpers.isElementVisibleWithTimeout(ANIMAL_CONTACT_LABEL, 1));
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/EpidemiologicalDataCaseSteps.java:660]: same

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java [hunk 5]
```diff
@@ -487,6 +487,18 @@ public CreateNewSampleSteps(
...
         });
+
+    When(
+        "I set Test Disease as ([^\"]*) in new pathogen result",
+        (String disease) -> webDriverHelpers.selectFromCombobox(TESTED_DISEASE_COMBOBOX, disease));
+
+    When(
+        "I check if Type of test in new pathogen results has no ([^\"]*) option",
+        (String typeOfTest) -> {
+          softly.assertFalse(
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java:498]: same

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

**Comment by pk-sgent** [comment]
after rebase + fixing 
image

**Pull request merged**